easybuild/easyconfigs/c/CVXOPT/CVXOPT-1.1.9-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = "PythonPackage"
+
+name = 'CVXOPT'
+version = '1.1.9'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://cvxopt.org'
+description = """CVXOPT is a free software package for convex optimization based on the Python programming language.
+ Its main purpose is to make the development of software for convex optimization applications straightforward by
+ building on Python's extensive standard library and on the strengths of Python as a high-level programming language.
+"""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+toolchainopts = {'pic': True}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['CVXOPT-1.1.9-fix-setup-py.patch']
+checksums = [
+    '8f157e7397158812cabd340b68546f1baa55a486ed0aad8bc26877593dc2983d',  # cvxopt-1.1.9.tar.gz
+    '89058d53c55dcdc94a9b7a6ea8c6a6e78a693210d1d27a4178f68926ae5099e3',  # CVXOPT-1.1.9-fix-setup-py.patch
+]
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SuiteSparse', '5.6.0', '-METIS-5.1.0'),
+    ('GSL', '2.6'),
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+preinstallopts = 'CVXOPT_BUILD_FFTW=1 CVXOPT_BUILD_GSL=1 CVXOPT_BLAS_EXTRA_LINK_ARGS="$LIBLAPACK" '
+preinstallopts += 'CVXOPT_FFTW_EXTRA_LINK_ARGS="$LIBFFT" CVXOPT_SUITESPARSE_SRC_DIR=$EBROOTSUITESPARSE '
+
+installopts = '--no-binary cvxopt'
+
+sanity_check_commands = ['nosetests']
```

Comment:
Sanity check commands are run from the installation directory, which in this case leads to:

```
Ran 0 tests in 0.001s
```

It's easy to pass the tests like this (and stupid that `nosetests` doesn't result in an error when there are no tests...).

Fixed in 938e2cf589